guides.md

```diff
@@ -26,6 +27,45 @@ our values.
 - [Voicemail](#voicemail)
 - [References](#references)
 
+## GitHub
+
+## Managing Labels
+
+We have _synced repos_ configured that mirror all labels from the root `hyphacoop/organizing`,
+on each change to the root repo's labels.
+Though some of the below steps are more complicated than we'd ideally like,
+this is because the automation is cautious and won't delete in-use labels,
```

Comment:
It's not our code here, just a `prune` setting on the tool that acts conservatively by not deleting labels that have even one issue using it in the repo. So `closed` and `open` are treated the same, but it shouldn't affect the instructions anyhow.

How about:
```suggestion
this is because the automation is cautious and won't delete _any_ in-use labels (regardless of open/closed state),
```

Reply:
Yea I suspected, so when I went thru it I also did for the closed ones, but bc it's an extra click to reveal closed issues so it's not obvious.

Reply:
Makes sense. Better to have no open questions on your mind when going through a guide :)

Does the change hint at that enough? Or any other suggestion?